google/cloud/forseti/services/inventory/crawler.py

```diff
@@ -286,6 +286,77 @@ def update(self, resource):
             raise
 
 
+def _api_client_factory(storage, config, parallel):
+    """Creates the proper initialized API client based on the configuration.
+
+    Args:
+        storage (object): Storage implementation to use.
```

Comment:
Is there any reason we are using object as the type instead of the actual Storage class (same for others)?

Reply:
The appropriate class isn't imported in this code. This needs to be cleaned up across the board and should be part of the work to move to pytype (#1938)